src/app/shared/components/external-cluster-credentials/provider/aks/template.html

```diff
@@ -22,6 +22,10 @@
            [formControlName]="Controls.TenantID"
            autocomplete="off"
            required>
+    <mat-hint>
+      Enter Azure Tenant ID.To create one
```

Comment:
```suggestion
      Enter Azure Tenant ID. To create one
```
Please also fix this in other changes in this PR below.